app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+WEATHER_BY_CITY_URL = "https://api.weatherapi.com/v1/current.json"
```

Comment:
not the best name, you could filter it by latitude, by city, by country...

Reply:
Agree with you
Thank